website/docs/wallet/withdraw-validator.md

```diff
@@ -0,0 +1,404 @@
+
+```
+python ./staking_deposit/deposit.py generate-bls-to-execution-change
+Please choose your language ['1. العربية', '2. ελληνικά', '3. English', '4. Français', '5. Bahasa melayu', '6. Italiano', '7. 日本語', '8. 한국어', '9. Português do Brasil', '10. român', '11. Türkçe', '12. 简体中文']:  [English]: english
+
+Please choose the (mainnet or testnet) network/chain name ['mainnet', 'goerli', 'sepolia', 'zhejiang']:  [mainnet]: zhejiang
+
+Please enter your mnemonic separated by spaces (" "). Note: you only need to enter the first 4 letters of each word if you'd prefer.: 
+bike shoe attitude violin fun life punch enhance attend bright voyage wheel clutch taxi high health siren jealous tell female upon firm manual wage
+
+Please enter the index (key number) of the signing key you want to use with this mnemonic. [0]: 0
+
+Please enter a list of the validator indices of your validator(s). Split multiple items with whitespaces or commas.: 8
+
+Please enter a list of the old BLS withdrawal credentials of your validator(s). Split multiple items with whitespaces or commas.: 00a6bd30000296e9c9f5823b09e689ff0bc0b1bea1d256caab9a5f213a226b33
+
+Please enter the 20-byte execution address for the new withdrawal credentials. Note that you CANNOT change it once you have set it on chain.: 0x9B984D5a03980D8dc0a24506c968465424c81DbE
+
+**[Warning] you are setting an Eth1 address as your withdrawal address. Please ensure that you have control over this address.**
+
+Repeat your execution address for confirmation.: 0x9B984D5a03980D8dc0a24506c968465424c81DbE
+
+**[Warning] you are setting an Eth1 address as your withdrawal address. Please ensure that you have control over this address.**
+
+Success!
+Your SignedBLSToExecutionChange JSON file can be found at: /home/me/Desktop/code/python/staking-deposit-cli/bls_to_execution_changes
+```
+
+### Step 3: Verify your output
+
+Once you complete the above, you’ll have a file contained in the `bls_to_execution_changes/` folder of your [staking-deposit-cli](https://github.com/ethereum/staking-deposit-cli). It will represent a list of BLS to execution messages that have been signed with your private keys and are ready to submit to Ethereum. Here’s what a sample file of these looks like:
+
+```
+[
+	{
+    "message": {
+      "validator_index": "838",
+      "from_bls_pubkey": "0xb89bebc655569726a318c8e9971bd3144497c61aea4a6578a7a4f94b547dcba5bac16a89108b6b6a1fe3695d1a874a0b",
+      "to_execution_address": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0a"
+    },
+    "signature": "0xa42103e15d3dbdaa75fb15cea782e4a11329eea77d155864ec682d7907b3b70c7771960bef7be1b1c4e08fe735888b950c1a22053f6049b35736f48e6dd018392efa3896c9e427ea4e100e86e9131b5ea2673388a4bf188407a630ba405b7dc5"
+  },
+	{
+    "message": {
+      "validator_index": "20303",
+      "from_bls_pubkey": "0xb89bebc699769726a502c8e9971bd3172227c61aea4a6578a7a4f94b547dcba5bac16a89108b6b6a1fe3695d1a874a0b",
+      "to_execution_address": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b"
```

Comment:
The `to_execution_address` field may be fine if it's the same and probably we want to use the same address on many different keys. I'm only guessing this will be the default for home stakers.

Reply:
should I change the example in some way?